### PR TITLE
fix: Description List - Update README

### PR DESCRIPTION
**Pull request by jameswklassen**
## Description
Update the README for `d2l-dl-wrapper`

### CODE REVIEW

File: components/description-list/README.md [hunk 1]
```diff
@@ -10,16 +10,20 @@ A Description List displays information in a vertical list of key-value pairs. C
 ```
 
 ## Best Practices
+Use a Description List to display static information that pairs property names and values. In the standard HTML description list `<dl>` element these are called terms `<dt>` and definitions `<dd>`.
```
Comment (jameswklassen) [inline components/description-list/README.md:13]: cc: @geurts 

This is the first time we've had text preceding the best practices.

This surfaced a minor spacing issue which I've tried addressing in https://github.com/BrightspaceUI/documentation/pull/1048.

image

File: components/description-list/README.md [hunk 2]
```diff
@@ -60,9 +67,9 @@ When the component width is less than the `breakpoint` property, the side-by-sid
 
 ## Description List Wrapper [d2l-dl-wrapper]
 
-The `d2l-dl-wrapper` can be combined with `descriptionListStyles` to apply styling and resize behavior to native `dl` elements.
+The `d2l-dl-wrapper` component can be combined with `descriptionListStyles` to apply styling and resize behavior to native `dl` elements.
 
-<!-- docs: demo live name:d2l-test-dl -->
+<!-- docs: demo live name:d2l-test-dl display:block -->
```
Comment (oelhanafey) [inline components/description-list/README.md:72]: Does the `display: block` do anything? I haven't seen this used before

File: components/description-list/README.md (hunk 1 above)
Comment (oelhanafey) [reply components/description-list/README.md:13]: I wonder if the text in _Best Practices_ would fit better under another section. I'll let @geurts chime in on that.

**Review by oelhanafey — COMMENTED** [review]
`feat:` will trigger a minor version bump for the core repo. For docs changes, we often use `docs:` which doesn't trigger a release at all or if you need the docs to update sooner you can trigger a patch version bump with `fix:`.

File: components/description-list/README.md (hunk 2 above)
Comment (jameswklassen) [reply components/description-list/README.md:72]: Yeah, this changes the `display` selector in the demo (defaults to flex). This felt like the cleanest way to resolve some of the resize issues I was seeing in the live demo.

**Comment by jameswklassen** [comment]
> `feat:` will trigger a minor version bump for the core repo. For docs changes, we often use `docs:` which doesn't trigger a release at all or if you need the docs to update sooner you can trigger a patch version bump with `fix:`.

Yeah I could do `docs:`, but I think a new version needs to be released for daylight to pick it up (?).

File: components/description-list/README.md (hunk 2 above)
Comment (oelhanafey) [reply components/description-list/README.md:72]: oo good to know, thanks!

**Comment by oelhanafey** [comment]
> Yeah I could do `docs:`, but I think a new version needs to be released for daylight to pick it up (?).

Yeah if you want to trigger a release, then `fix:` would be better since it just triggers a patch version bump instead of a minor version bump

**Review by dbatiste — APPROVED** [review]

**Review by oelhanafey — APPROVED** [review]

**Review by eKoopmans — APPROVED** [review]

**Review by dbatiste — APPROVED** [review]

**Pull request merged**